README.md

```diff
@@ -15,7 +15,11 @@ A simple command handler for [Hikari](https://github.com/hikari-py/hikari).
 ## Features
  - Simple and intuitive API.
  - Slash, user, and message commands.
- - Error handling.
+ - Error handling for commands, events, and autocomplete.
+ - Command groups.
+ - Hooks to run function before a command (or any command from a group!)
```

Comment:
`function` -> `functions` or `a function`. Also did we add support for calling hooks *after* a command?

Reply:
kk